### PR TITLE
fixing Issue #4335 - Not quoting column name when doing a time series

### PR DESCRIPTION
**Pull request by davelpz**
Fixed issue as discussed in Issue #4335 comments.

### CODE REVIEW

**Comment by rumbin** [comment]
Will this PR respawn issue #3844? See also #4046 for reference.

File: superset/connectors/sqla/models.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def get_time_filter(self, start_dttm, end_dttm):
     def get_timestamp_expression(self, time_grain):
         """Getting the time component of the query"""
         expr = self.expression or self.column_name
-        if not self.expression and not time_grain:
+        if not self.expression and (not time_grain or time_grain == u'Time Column'):
```
Comment (mistercrunch) [inline superset/connectors/sqla/models.py]: `u'foo'` isn't python3 compatible I think. I'm surprised that label-looking `Time Column` is a possible value here, somehow it should be `time_column` or `None` instead.

**Review by mistercrunch — CHANGES_REQUESTED** [review]

File: superset/connectors/sqla/models.py (hunk 1 above)
Comment (davelpz) [reply superset/connectors/sqla/models.py]: Yea, it's u'Time Column' not None or something else

Comment (davelpz) [reply superset/connectors/sqla/models.py]: I looked up the python 3 docs, the u'foo' is valid in python 3

**Comment by kstan79** [comment]
hi any solution regarding this? During I'm evaluate superset via my sample mysql data warehouse, notice that lot of charts cannot be generate due to no backquote column name. Example like Sum(Sales Amount) cause sql error.

**Comment by mistercrunch** [comment]
@kstan79 I think this is fixed on master, can you confirm?

**Pull request closed without merging**